documentation/release-notes-2-8.md

```diff
@@ -1,26 +1,23 @@
 ---
 layout: default
-title: Relax-and-Recover Release Notes
+title: Release Notes for Relax-and-Recover Version 2.8
 ---
 
+This document contains the latest release notes for the Open Source Software project Relax-and-Recover.
 
-# Release Notes for Relax-and-Recover Version 2.8
+Relax-and-Recover website: [relax-and-recover.org](https://relax-and-recover.org/)
```

Comment:
see
https://github.com/rear/rear/issues/3238#issuecomment-2531329996
why I intentionally had changed the links
to plain text for the link label
and URLs within angle brackets

Reply:
Ah!
I missed that now there is a
```
how to use Lynx to convert to proper text file
without manual processing
```